### PR TITLE
fix: move crypto org into cosmos coin module

### PR DESCRIPTION
**Pull request by qperrot**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Move `cryto_org` into cosmos coin module.
Change `stakingDocUrl`

https://github.com/user-attachments/assets/318976ae-cbff-4a2f-af55-5c53beae043c

### ❓ Context

- **JIRA or GitHub link**: [LIVE-13739](https://ledgerhq.atlassian.net/browse/LIVE-13739)
-  [LIVE-15018](https://ledgerhq.atlassian.net/browse/LIVE-15018)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13739]: https://ledgerhq.atlassian.net/browse/LIVE-13739?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-15018]: https://ledgerhq.atlassian.net/browse/LIVE-15018?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ypolishchuk-ledger — DISMISSED** [review]

**Review by Canestin — CHANGES_REQUESTED** [review]
#### Manual tests on Ledger Live

1. Add Account Modal:  
   - Letters not displaying during search  
   - "Cronos POS Chain" missing  
	
	<br>
	
	image

File: apps/ledger-live-desktop/src/newArch/features/MemoTag/constants.ts [hunk 1]
```diff
@@ -4,8 +4,6 @@ export const MEMO_TAG_COINS: string[] = [
   "cosmos",
   "hedera",
   "injective",
-  "crypto_org",
-  "crypto_org_croeseid",
```
[the remark this answers is not in the document]
Comment (qperrot) [reply apps/ledger-live-desktop/src/newArch/features/MemoTag/constants.ts:8]: For `crypto_org` you need to use `Cronos POS Chain`

[the remark this answers is not in the document]
Comment (qperrot) [reply apps/ledger-live-desktop/src/newArch/features/MemoTag/constants.ts]: @Canestin I added memotag back !
But I can not reproduce your error about `Cronos POS Chain` missing and `Letters not displaying during search`

**Review by Canestin — DISMISSED** [review]

**Review by Wozacosta — APPROVED** [review]
gj 🚀

**Review by Canestin — DISMISSED** [review]
GG :rocket:

**Review by Canestin — DISMISSED** [review]

File: apps/ledger-live-mobile/src/components/RootNavigator/types/SendFundsNavigator.ts
Comment (themooneer) [inline apps/ledger-live-mobile/src/components/RootNavigator/types/SendFundsNavigator.ts:1]: if the feature flag is off , we may need to have the state of the app with those deleted files no ?

File: libs/ledger-live-common/package.json [hunk 2]
```diff
@@ -127,7 +127,9 @@
     "@celo/utils": "^7.0.0",
     "@celo/wallet-base": "^6.0.3",
     "@celo/wallet-ledger": "^6.0.3",
-    "@crypto-org-chain/chain-jslib": "1.1.2",
```
Comment (themooneer) [inline libs/ledger-live-common/package.json:130]: you need to delete it from `libs/ledger-live-common/dependencies.md` table  too

**Review by themooneer — APPROVED** [review]
Verification tests LLD: 
- App startup ✅
- Send Flow with cosmos ✅

Verification tests LLM:
- App startup ✅

Verified bundle size 36.7mb  ✅

LGTM

**Review by Wozacosta — APPROVED** [review]

**Pull request merged**